Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -115,6 +117,8 @@
             return @"INVALID_APPLE_SUBSCRIPTION_KEY";
         case RCIneligibleError:
             return @"INVALID_OFFER";
```

Comment:
Hmm, unrelated but I think this should be @"INELIGIBLE_ERROR", can you change it?

Reply:
sure thing!